invest/templates/invest/high-potential-opportunity-detail.html

```diff
@@ -0,0 +1,231 @@
+                </Ul>
+            </div>
+            <div class="column-one-half">
+                {% if page.proposition_two_video %}
+                    {{ page.proposition_two_video }}
+                {% else %}
+                    <img src="{{ page.proposition_two_image.url }}" />
+                {% endif %}
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="competative-advantages">
+    <div class="container">
+        <div class="grid-row">
+            <h2 class="heading-large">{{ page.competative_advantages_title }}</h2>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_one_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_one|add_export_elements_classes }}
+                    <div class="clearfix"></div>
+                </div>
+            </div>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_two_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_two|add_export_elements_classes }}
+                </div>
+            </div>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_three_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_three|add_export_elements_classes }}
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="testimonial">
+    <div class="container">
+        <div class="grid-row">
+            {{ page.testimonial|add_export_elements_classes }}
+        </div>
+    </div>
+</section>
+
+<section id="company-list">
+    <div class="container">
+        <div class="grid-row">
+            <div>
+                {{ page.companies_list_text|add_export_elements_classes }}
+            </div>
+            <ul>
+                <li class="company-list-first-line">
+                    <img src="{{ page.companies_list_item_image_one.url }}" />
+                    <img src="{{ page.companies_list_item_image_two.url }}" />
+                    <img src="{{ page.companies_list_item_image_three.url }}" />
+                    <img src="{{ page.companies_list_item_image_four.url }}" />
+                </li>
+                <li>
+                    <img src="{{ page.companies_list_item_image_five.url }}" />
+                    <img src="{{ page.companies_list_item_image_six.url }}" />
+                    <img src="{{ page.companies_list_item_image_seven.url }}" />
+                    <img src="{{ page.companies_list_item_image_eight.url }}" />
+                </li>
+            </ul>
+        </div>
+    </div>
+</section>
+
+<section id="case-studies">
+    <div class="container">
+        <div class="grid-row">
+            <h2 class="heading-xlarge">{{ page.case_study_list_title }}</h2>
```

Comment:
make sure to put headings outside of `grid-row` divs otherwise the outer margins won't line up (grids have a negative margin)